cfgov/v1/models/images.py

```diff
@@ -103,6 +103,27 @@ class CFGOVRendition(AbstractRendition):
     def alt(self):
         return self.image.alt
 
+    def get_orientation(self):
+        orientation = 'square'
+        if self.is_portrait:
+            orientation = 'portrait'
+        elif self.is_landscape:
+            orientation = 'landscape'
+
+        return orientation
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3161/commits/59f9f88b9b1c6b47d1da8d35f78cb0327db87c19